src/fetch_receipts.py

```diff
@@ -9,8 +9,14 @@
 import numpy as np
 import pandas as pd
 
+from itertools import islice
+from multiprocessing import Pool
 
-class Receipts:
+
+class Receipts(object):
```

Comment:
AFAIK there's no need to inherit from `object` in Python 3 — is there a specific reason to do this here?